spec/controllers/scim_rails/scim_users_controller_spec.rb

```diff
@@ -529,7 +529,7 @@
         user = company.users.first
         expect(user.archived?).to eq false
 
-        patch :patch_update, params: patch_params(id: 1)
+        patch :patch_update, params: patch_params(id: 1), as: :json
```

Comment:
We actually had a spec testing that you could de provision a user via the patch endpoint and it was passing, even though the functionality was broken. The reason was because we weren't treating these requests as JSON in the controller spec even though that is how they come through IRL, so instead of `"active": false` it was coming in as `"active": "false"` in the test and therefore passing.

I updated the entire spec to have `as: :json` as a result of finding that to be sure the test mimics what we expect from real requests.